### DISCUSSION TITLE
Ansel+DT parallel on Win

### DISCUSSION THREAD

**Discussion by mediatechnologic** [aurelienpierreeng/ansel#67] [Q&A] [answered]
How to start Ansel with already installed DT on windows?
I know about the config filles and location under c:\users\… but I need instructions what to do. 
Is there any web page about this topic existing?

Of course I know about https://ansel.photos/en/doc/cli/ansel/
but which of all these parameters are important to exclude any risk. 

My plan is not yet to use Ansel with DT on the same image directories (XMP colission).
May be there is no risk at all with this?

Thank you and a good New Year 2023
Rudy

**Comment by mediatechnologic** [comment, marked as answer]
I took the risk (which was in fact not existing) and installed paralell to DT.
Perfect, the two relevant directories in Win are completely independant:
- c:\Users\USER\AppData\Local\ansel\
- c:\Users\USER\AppData\Local\darktable\

I do an automatic backup of multiple of such config-directories 
by creating symbolic links in one other directory to above directories.
Then, this one directory (with the symbolic links) is backup manually or regularly.

There is one (currently empty) directory at c:\Users\USER\ansel\

I am really totally impressed by Ansel. Thank you very much Aurelien!
The GUI look so clean and concentrates on editing.
Other tools can do the DAM work and other tasks.
A toolchain of optimized, diferent programs is better than one overloaded
https://fr.wiktionary.org/wiki/eierlegende_Wollmilchsau
https://en.wiktionary.org/wiki/eierlegende_Wollmilchsau

**Comment by aurelienpierre** [comment]
Thanks for the feedback !
